packages/atomic/cypress/e2e/result-list/result-components/quickview.cypress.ts

```diff
@@ -0,0 +1,145 @@
+import {generateComponentHTML} from '../../../fixtures/fixture-common';
+import {TestFixture} from '../../../fixtures/test-fixture';
+import * as CommonAssertions from '../../common-assertions';
+import {addFacet} from '../../facets/facet/facet-actions';
+import {addSearchBox} from '../../search-box/search-box-actions';
+import {addResultList, buildTemplateWithSections} from '../result-list-actions';
+import {
+  quickviewComponent,
+  QuickviewSelectors,
+  QuickviewModalSelectors,
+} from './quickview-selectors';
+
+const addQuickviewInResultList = () =>
+  addResultList(
+    buildTemplateWithSections({
+      actions: generateComponentHTML(quickviewComponent),
+    })
+  );
+
+const openModal = () => {
+  cy.intercept('**/build/iconButton*').as('quickviewResources');
```

Comment:
This intercept/wait logic is needed to workaround a bug currently in atomic-modal, where if it gets opened "too fast" before it has a chance to load internal resources, it fails to perform it's job properly.

It's a bug that's currently only apparent for Cypress, since here we are interested in opening the Quickview ASAP, which causes flakyness in the test execution.